### DISCUSSION TITLE
Stackable snackbar with close button

### DISCUSSION THREAD

**Discussion by salucien** [Megabit/Blazorise#2469] [Q&A]
Hello, 
Is there a way to display a close button for stackable snackBar ?
I tried this : 
```
snackbarStack.PushAsync("Merci d'entrer au moins une référence",SnackbarColor.Danger, (options) => {  options.CloseButtonText = "OK"; options.ShowCloseButton = true; });
```
but "OK " is not displayed

**Comment by stsrki** [comment]
That should work as far as I remember. Unfortunately I cannot check as I l'm not near my pc this weekend. As soon as I get back I will look into it.
